src/public/modules/forms/base/directiveViews/submit-form.directive.view.html

```diff
@@ -11,7 +11,7 @@
       auth-type="form.authType"
       is-admin-preview="false"
       is-template="form.isTemplate"
-      form-login="formLogin(authType, rememberMe)"
+      form-login="formLogin(isPersistentLogin)"
```

Comment:
is this set on the `scope`? I don't think it is, which means it is always `undefined`

Reply:
ah nvm, I see that it's using `&` binding

Reply:
angularjs strikes again

Reply:
the binding thing is still very trippy tbh. just leaving [this](https://www.codelord.net/2016/05/13/understanding-angulars-and-binding/) here for the weird & binding thing in case anyone needs it (i.e, future me)